### PR TITLE
Make week header optionally not scrollable

### PR DESCRIPTION
**Pull request by anvukelic**
Issue https://github.com/boguszpawlowski/ComposeCalendar/issues/105

### CODE REVIEW

File: library/src/main/java/io/github/boguszpawlowski/composecalendar/Calendar.kt [hunk 1]
```diff
@@ -73,6 +74,7 @@ public fun SelectableCalendar(
   today: LocalDate = LocalDate.now(),
   showAdjacentMonths: Boolean = true,
   horizontalSwipeEnabled: Boolean = true,
+  scrollWeekDays: Boolean = true,
```
Comment (boguszpawlowski) [inline library/src/main/java/io/github/boguszpawlowski/composecalendar/Calendar.kt:77]: In order to make it more similar to the other boolean flag, I would call it:
`weekDaysScrollEnabled`

File: library/src/main/java/io/github/boguszpawlowski/composecalendar/month/MonthContent.kt [hunk 2]
```diff
@@ -66,25 +68,36 @@ internal fun <T : SelectionState> MonthPager(
...
-        showAdjacentMonths = showAdjacentMonths,
-        selectionState = selectionState,
-        currentMonth = monthListState.getMonthForPage(index),
-        today = today,
-        daysOfWeek = daysOfWeek,
-        dayContent = dayContent,
-        weekHeader = weekHeader,
-        monthContainer = monthContainer
+  Column(modifier = Modifier.fillMaxWidth()) {
+    if (!scrollWeekDays) {
```
Comment (boguszpawlowski) [inline library/src/main/java/io/github/boguszpawlowski/composecalendar/month/MonthContent.kt:72]: I would flip the condition to not have negation in there, as it is harded to read. 
```suggestion
    if (scrollWeekDays) {
```

**Review by boguszpawlowski — APPROVED** [review]
Overall, the code LGTM. After the small adjustments mentioned in the comments, I will merge it.

**Comment by boguszpawlowski** [comment]
Please also re-generate the API contract: `./gradlew apiDump`

File: library/src/main/java/io/github/boguszpawlowski/composecalendar/month/MonthContent.kt (hunk 2 above)
Comment (anvukelic) [reply library/src/main/java/io/github/boguszpawlowski/composecalendar/month/MonthContent.kt:72]: There's no else block here - do you want me to switch it with .not() call so it becomes easier to read?

**Review by boguszpawlowski — APPROVED** [review]
I have tested the code, and it works fine, we can merge it after you apply this final comment.

**Comment by anvukelic** [comment]
This is not merged, should I do something else?

File: library/src/main/java/io/github/boguszpawlowski/composecalendar/Calendar.kt [hunk 3]
```diff
@@ -73,6 +74,7 @@ public fun SelectableCalendar(
   today: LocalDate = LocalDate.now(),
   showAdjacentMonths: Boolean = true,
   horizontalSwipeEnabled: Boolean = true,
+  weekDaysScrollEnabled: Boolean = true,
```
[the remark this answers is not in the document]
Comment (anvukelic) [reply library/src/main/java/io/github/boguszpawlowski/composecalendar/Calendar.kt:77]: @boguszpawlowski done 👍

**Comment by boguszpawlowski** [comment]
@anvukelic Merged, thanks once again for your contribution. I will update libraries and then release next version. Meanwhile if you need this functionality, you can use snapshot version.

**Pull request merged**